### PR TITLE
React capstone project - Country metrics

### PR DESCRIPTION
**Pull request by SFarida**
### This PR contains the following:
- a home page that displays countries
- a countryDetails page to display the selected country's details
- a search field to search a country
- test cases

### CODE REVIEW

File: src/components/CountryDetails.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    );
+  }
+
+  return (
+    <div>
+      {content}
+      <Countries />
+    </div>
+  );
+};
```
Comment (Favourezeugwa) [inline src/components/CountryDetails.js]: - Great Job displaying the details of each country 💪🏽 
- I would like to kindly suggest that this page sticks to just the details of a selected country alone. You really don't need the list of all countries on the details page. Can we fix that?

Here is what I mean 👇🏽 

| Yourdetails page  |
| :---:   | 
|Screenshot 2023-03-17 at 18 32 50  |

File: src/components/Countries.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            <Country
+              key={country.countryCode}
+              name={country.name}
+              code={country.countryCode}
+            />
+          ))
+        }
+      </div>
+    </section>
+  );
```
Comment (Favourezeugwa) [inline src/components/Countries.js:32]: - Great job mapping a few countries in your project. 
- I would like to kindly suggest that we stick to the design guidelines on behance as much as possible. On behance, the layout uses grid on mobile screen, with two countries next to each other. But in your project, each country is displayed singly like a stack. Can we fix this? 🌸 

Here is what i can see compared to Figma 👇🏽 
| Your Design  |Behance Design |
| :---:   | :---: | 
| Screenshot 2023-03-17 at 18 22 00 |Screenshot 2023-03-17 at 18 42 03  |

**Review by Favourezeugwa — CHANGES_REQUESTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @SFarida , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

close lady

## Highlights 💯 
- No Linter errors ✔️
- Descriptive Pull request summary✔️
- Good commit messages ✔️
- Professional Readme file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Countries.js (hunk 2 above)
Comment (SFarida) [reply src/components/Countries.js:32]: Hello @Favourezeugwa thank you for the remark, I think the app onrender was not up to date, I just updated it. Please is it possible for you to check and give your comments on that design.

Comment (SFarida) [reply src/components/Countries.js:32]: Thank you @Favourezeugwa did the necessary corrections and update the code on render too.

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @SFarida, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Git Flow. :heavy_check_mark: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**